### DISCUSSION TITLE
Packages/com.unity.recorder/manual/KnownIssues

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#647] [Notes]
# Packages/com.unity.recorder/manual/KnownIssues

https://docs.unity3d.com/Packages/com.unity.recorder@latest/index.html?subfolder=%2Fmanual%2FKnownIssues.html

**Comment by lazlo-bonin** [comment]
This third party package offers Wwise audio recording: 
https://github.com/nehvaleem/com.undefined.wwise-recorder
